swagger.json

```diff
@@ -2117,6 +2117,170 @@
                 ]
             }
         },
+        "/organizers/{cdbid}/website": {
```

Comment:
Volgens de huidige Swagger docs noemt dit property eigenlijk "url" in de JSON-LD van organizer. Misschien dan beter hier ook "url" gebruiken ipv "website" om consistent te blijven?